loudspeaker.podspec

```diff
@@ -9,7 +9,7 @@
 
 Pod::Spec.new do |s|
   s.name             = "loudspeaker"
-  s.version          = "0.1.2"
+  s.version          = "0.1.3"
```

Comment:
I am sorry. Changing podspec in separate commit, but maybe I misunderstood you. How to I can fix this?